sw/host/opentitanlib/src/transport/ti50emulator/gpio.rs

```diff
@@ -263,6 +266,11 @@ impl GpioPin for Ti50GpioPin {
         self.update_and_notify(|host_state| host_state.value = value)
     }
 
+    fn reset(&self) -> Result<()> {
+        let state = resolve_state(" ", Logic::WeakZero, self.default_level).unwrap();
```

Comment:
Please could you add a comment here stating that the `name` is `" "` because the error case is never hit?

Or maybe you could pass `"unreachable"` as the name